annotations/views.py

```diff
@@ -900,25 +902,56 @@ def user_details(request, userid, *args, **kwargs):
     :HTTPResponse:
         Renders an user details view based on user's authentication status.
     """
-
     user = get_object_or_404(VogonUser, pk=userid)
-    if request.user.is_authenticated() and request.user.id == userid:
-        template = loader.get_template('annotations/user_details.html')
-        context = RequestContext(request, {
-            'user': request.user,
-            'detail_user': user,
-        })
+    if request.user.is_authenticated() and request.user.id == int(userid):
+        return HttpResponseRedirect("/accounts/settings/")
     else:
         textCount = Text.objects.filter(addedBy=user).count()
         textAnnotated = Text.objects.filter(annotators=user).distinct().count()
         relation_count = user.relation_set.count()
+        start_date = datetime.datetime.now() + datetime.timedelta(-60)
+
+        #Count annotations for user by date
+        annotation_by_user = Relation.objects.filter(createdBy = user, created__gt = start_date)\
+        .extra({'date' : 'date(created)'}).values('date').annotate(count = Count('created'))
```

Comment:
If you're spanning multiple lines, use some indentation to make the code easier to read. E.g. 

```python
annotation_by_user = Relation.objects.filter(createdBy = user, created__gt = start_date)\
    .extra({'date' : 'date(created)'}).values('date').annotate(count = Count('created'))
```